src/api/test/unit/schema_test.rb

```diff
@@ -19,7 +19,6 @@ class SchemaTest < ActiveSupport::TestCase
       if io
         testresult = io.read
         io.close
-        # rubocop:disable Style/NumericPredicate
         assert $? == 0, "#{testfile} does not validate against #{f} -> #{testresult}"
         # rubocop:enable Style/NumericPredicate
       end
```

Comment:
This `rubocop:enable` line can also be deleted.

Reply:
this was automatically corrected by rubocop, but yes :+1:

Reply:
I sent an issue to Rubocop regarding this :wink: